src/libAtomVM/module.c

```diff
@@ -201,13 +213,24 @@ Module *module_new_from_iff_binary(GlobalContext *global, const void *iff_binary
     mod->fun_table = beam_file + offsets[FUNT];
     mod->str_table = beam_file + offsets[STRT];
     mod->str_table_len = sizes[STRT];
-    mod->labels = calloc(ENDIAN_SWAP_32(mod->code->labels), sizeof(void *));
+    uint32_t num_labels = ENDIAN_SWAP_32(mod->code->labels);
+    mod->labels = calloc(num_labels, sizeof(void *));
```

Comment:
I didn't change this one because it is existing code and want to limit the scope of the PR (I made this change so I could get num_labels out into a variable), but I fixed the one 👇🏽